lib/team_api/joiner.rb

```diff
@@ -64,6 +64,7 @@ def team_members
       @team_members ||= team.map { |key, value| value unless key == 'private' }
         .compact
         .concat((team['private'] || {}).values)
+      @team_members
```

Comment:
This method should already return `@team_members` without this explicit statement. Does it not?